src/crab/ebpf_transformer.cpp

```
@@ -635,6 +641,16 @@ void EbpfTransformer::operator()(const Mem& b) {
     if (const auto preg = std::get_if<Reg>(&b.value)) {
         if (b.is_load) {
             do_load(b, *preg);
+            if (b.is_signed) {
+                Bin::Op op{};
```

Clang (14.0.6)

[warning] variable name 'op' is too short, expected at least 3 characters

(readability-identifier-length)